### PR TITLE
VMware: Add teaming policy for DVS Portgroup.

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py

##### ANSIBLE VERSION

```
2.5devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
@pdellaert Could you please take a look ?

File: lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py [hunk 1]
```diff
@@ -81,6 +81,17 @@
...
         required: False
         version_added: '2.5'
+    teaming_policy:
+        description:
+            - Dictionary which configures the different teaming values for portgroup.
+            - 'Valid attributes are:'
+            - '- C(load_balance_policy) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
+            - '   - choices: [ loadbalance_ip, loadbalance_srcmac, loadbalance_srcid, failover_explicit]'
+            - '- C(inbound_policy) (bool): Indicate whether or not the teaming policy is applied to inbound frames as well. (default: False)'
+            - '- C(notify_switches) (bool): Indicate whether or not to notify the physical switch if a link fails. (default: False)'
```
Comment (pdellaert) [inline lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py]: When creating a portgroup in the UI, by default, notify_switches is yes. I think we should stay in line with that.

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py]: Done.

**Review by pdellaert — APPROVED** [review]
shipit

Looks great! Thanks :)

**Comment by Akasurde** [comment]
@pdellaert Thanks for review.

**Pull request merged**